honeybee/model.py

```diff
@@ -1349,6 +1375,21 @@ def to_dict(self, included_prop=None, triangulate_sub_faces=False):
 
         return base
 
+    @staticmethod
+    def to_json(hb_dict, folder_path, name="unnamed"):
+        """Writes a Honeybee model dictionary to hbjson
+
+        Args:
+            hb_dict (dictionary): A Honeybee model in the form of a dictionary
+            folder_path (A text string): Folder location where json will be written
+            name (str, optional): File name. Defaults to "unnamed".
+        """
+        file_name = '{}.hbjson'.format(name)
```

Comment:
What if the name already ends with `.hbjson`?  The Wea.write() method is probably a good example for you to follow:
https://github.com/ladybug-tools/ladybug/blob/master/ladybug/wea.py#L968